loops_lab.js

```diff
@@ -0,0 +1,202 @@
+//console.log( "---------")
+
+for ( let i = 0; i <= 100; i ++){
+  if ( i % 3 === 0){
+    console.log("Fizz")
+  } else if ( i % 5 === 0){
+      console.log( "Buzz")
+  } else {
+      console.log(i)
+  }
+  if ( (i % 3 ===0 ) && (i % 5 ===0)){
+    console.log("FizzBuzz")
+  }
+}
+// ### Bonus:
+//
+// 1. Write a program that would log the [lyrics of the song 99 Bottles of Beer](http://www.99-bottles-of-beer.net/lyrics.html). This is the first verse of the song:
+//
+//   ```
+//   99 bottles of beer on the wall, 99 bottles of beer.
+//   Take one down, pass it around, 98 bottles of beer on the wall.
+//   ```
+//
+//   This verse is repeated, each time with one less bottle, until the number of bottles is 0. When the number of bottles is 2, the verse is:
+//
+//   ```
+//   2 bottles of beer on the wall, 2 bottles of beer.
+//   Take one down, pass it around, 1 bottle of beer on the wall.
+//   ```
+//
+//   In the last line, the word bottles (plural), is  replaced with bottle (singular)
+//
+//   When the number of bottles is 1, the verse is:
+//
+//   ```
+//   1 bottle of beer on the wall, 1 bottle of beer.
+//   Take one down, pass it around, No more bottle of beer on the wall.
+
+
+for ( let beerBottles= 99; beerBottles >=0; beerBottles -=1) {
+    if (beerBottles !== 0 ){
+      console.log( beerBottles + " bottles of beer on the wall, " + beerBottles + " bottles of beer.")
+      console.log(" Take one down and pass it around, " + beerBottles + " bottles of beer on the wall. \n")
+    } else {
+      console.log("No more bottles of beer on the wall, no more bottles of beer.")
+      console.log("Go to the store and buy some more, " + beerBottles + " bottles of beer on the wall.")
+    }
+  }
+
+
+//___
+//
+// 2. Use the `assignGrade` function (given below). Write a function that logs each number from 60 - 100 along with its corresponding letter score.
+// Exp For each number from 81 to 90, log B, like so:
+//
+// ```js
+// 60 - F
+// ...
+// 81 - B
+// 82 - B
+// 83 - B
+// ...
+// 93 - A
+// 94 - A
+// ...
+// ```
+//
+// ```js
+function assignGrade(score) {
+    if (score > 90) {
+        return 'A';
+    } else if (score > 80) {
+        return 'B';
+    } else if (score > 70) {
+        return 'C';
+    } else if (score > 65) {
+        return 'D';
+    } else {
+        return 'F';
+    };
+};
+// ```
+// <details>
+// <summary>
+//   <b>Hint</b>
+// </summary>
+//   Explore this:
+
+for (let score = 60; score <= 100; score ++){
+       console.log(score + " - " + assignGrade(score))
+}
+
+//3. Given an integer N draw a square of N x N asterisks. Look at the examples.
+
+
+
+
+//   ```js
+  // let grade = assignGrade(80)
+  // console.log(grade)
+//   ```
+//   What is happening here? Can you explain it?
+// the the function assignGradeis being called and the value 80 is being placed insde as the score as, the program
```

Comment:
Overall good demonstration of for and while loops. Keep practicing and keep up the good work!